homeassistant/components/climate/eq3btsmart.py

```diff
@@ -81,24 +102,41 @@ def set_temperature(self, **kwargs):
         self._thermostat.target_temperature = temperature
 
     @property
-    def device_state_attributes(self):
-        """Return the device specific state attributes."""
-        return {
-            ATTR_MODE: self._thermostat.mode,
-            ATTR_MODE_READABLE: self._thermostat.mode_readable,
-        }
```

Comment:
Indeed, I forgot about /dev-state completely. They are there now and it seems to work, thanks.